### DISCUSSION TITLE
Hybrid Meetings

### DISCUSSION THREAD

**Discussion by ATLCOTech** [code4recovery/12-step-meeting-list#1281] [Q&A] [answered]
Hi Mark here from Atlanta Central Office. We have some meetings that are hybrid and noticed on the app that only the physical location is showing even though we have both an address and zoom code showing. 

On the website it is showing online and in person. Has there been any discussion about adding a hybrid meeting type so that the app picks up both?

**Comment by joshreisner** [comment]
hi Mark! i'm not seeing the 'normal' legacy TSML view at https://atlantaaa.org/meetings/ - typically one sees a green `Online` in the listing like on nyintergroup.org. (see screenshot). i think this could be due to the [customizations we discsused](https://github.com/code4recovery/12-step-meeting-list/discussions/1233) in this thread? 

<img width="1211" alt="Screenshot 2023-12-13 at 1 00 38 PM" src="https://github.com/code4recovery/12-step-meeting-list/assets/1551689/66ade09f-a922-4760-aede-069daf11ef00">

we've improved the UI quite a bit with our TSML UI appearance - this makes the way to join super clear with colorful chiclets:
<img width="1215" alt="Screenshot 2023-12-13 at 1 02 53 PM" src="https://github.com/code4recovery/12-step-meeting-list/assets/1551689/be47ae54-3ad3-4997-b01c-6ee30a46a555">

we think this is a better approach than adding an explicit "Hybrid" type for [reasons we have outlined here](https://github.com/code4recovery/tsml-ui#why-no-hybrid-type).

please note that atlanta central office would need to fix the missing conference_url issue before trying TSML UI, if that was of interest

**Reply by joshreisner** [reply, marked as answer]
@ATLCOTech when you were referring to the "app" i'm assuming above you were talking about your website

if you instead meant the meeting guide app, then that thread above, and in partcular this reply has the answer: https://github.com/code4recovery/12-step-meeting-list/discussions/1233#discussioncomment-7319815
